FocusOn/ViewModel/ProgressViewModel.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  ProgressViewModel.swift
+//  FocusOn
+//
+//  Created by Alexandra Ivanova on 12/07/2023.
+//
+
+import Foundation
+import SwiftUICharts
+import SwiftUI
+
+class ProgressViewModel: ObservableObject {
+    @Published var allGoals: [Goal]?
```

Comment:
The next iteration is to have this property `private` and to use `MockDataService` to input your test data to your unit test.

Your unit tests will look like:
```
func testSomething() {

    let mockDataService = MockDataService()
    mockDataService.allGoals = <your test data>

    let viewModel = ProgressViewModel(dataService: mockDataService)

    viewModel.executeMethodToTest()
}

```